src/psycopmlutils/timeseriesflattener/flattened_dataset.py

```diff
@@ -478,7 +493,20 @@ def flatten_temporal_values_to_df(
 
         df.rename({"val": full_col_str}, axis=1, inplace=True)
 
+        if is_fallback_prop_warning_threshold is not None and direction is "ahead":
+            prop_of_values_that_are_fallback = (
+                df[df[full_col_str] == fallback].shape[0] / df.shape[0]
+            )
+
+            if prop_of_values_that_are_fallback > is_fallback_prop_warning_threshold:
+                raise ValueError(
+                    f"""{prop_of_values_that_are_fallback*100}% of {full_col_str} contain the fallback value, indicating that it is unlikely to be a learnable feature.\n
```

Comment:
Primary focus is here 👍 

The rest is scaffolding as a consequence of implementing it.

Reply:
<3